.appsody-config.yaml

```diff
@@ -0,0 +1 @@
+stack: appsody/swift:0.1
```

Comment:
What is this???

Reply:
This is configuration for users who want to run Linux tests using Appsody. It is an open source project that IBM is leading and it would be good to offer first class support for it in the future.

Reply:
Hmmm.  What about the 99 other open-source development environment projects out there?  Are we going to support those too?

I'm not a fan.  I don't think we should be injecting this kind of thing into our SDKs.

Reply:
We don't necessarily have to support anything here. This is a convenience for contributors to the SDK to be able to use an open source package that IBM is behind in order to run their Linux tests in Docker in a no-configuration fashion. It doesn't put the onus on us to do anything else.

I think this could be a nice tool to use going forward, and it aligns nicely with how IBM is talking about Swift publicly on the server, but if this is a very contentious addition I will remove it.